### DISCUSSION TITLE
Pref file not saving (windows)

### DISCUSSION THREAD

**Discussion by Nylongbow** [iterate-ch/cyberduck#16359] [Q&A]
We've lost the ability for CD to save any settings within the preferences. Specifically the default editor. This means that in order to edit anything in a non-default editor (like sublime) you have to right click, edit with, more options, select sublime.

This occurred after updating over the summer.

This has been tested on machine with bot restricted student access as well as full admin privileges. The workflow has been dramatically slowed down.

Has anyone else seen this or can give me a hint on a possible fix?
thanks

**Comment by AliveDevil** [comment]
Do you exit Cyberduck normally, or do you kill it?

**Reply by Nylongbow** [reply]
exited normally. 
checking the cyberduck users config file we notice there is no XML entry for the default editor. 
Having a machine that was working we found the syntax for the editor and added it manually. (with cyberduck not running). 
saved, and reopened file to confirm the save. 

then as soon as we launch CD it seems to overwrite the file and remove the entry.
Also interestingly, we know this isn't a permission issue for the file. Not only because we edited it on the student profile logiun but also when turning on debug mode, we can confirm the debug flag being set in that file.

hoping some of this helps. The debug on isn't helpful as it never really crashes, just wont remember the editor default.

HTH

**Comment by AliveDevil** [comment]
Bug report created in #16362.

**Reply by Nylongbow** [reply]
Any update or resolution on this? workaround? 
the workflow for editing dozens of files x 60 students is becoming extremely cumbersome.

thanks.

**Reply by Nylongbow** [reply]
any idea when we can see this fix?
